lru/lru_test.go

```diff
@@ -114,10 +114,10 @@ func TestPurge(t *testing.T) {
 }
 
 func TestExpiry(t *testing.T) {
-	c := New(3, WithExpiry(100*time.Millisecond))
+	c := New(3, WithExpiry(time.Second))
```

Comment:
Was this change intentional? 100 milliseconds != 1 second.
The same goes for the below changes - 
```
time.Sleep(500 * time.Millisecond)

time.Sleep(700 * time.Millisecond)
```

Reply:
Yes. This is a time-based test.
The change to 1 second is to make sure it passed in 100% of the cases.